logstash-conf/my.conf

```diff
@@ -0,0 +1,28 @@
+input {
+  file {
+    path => "/var/log/pulse/collection-roller.log"
+    start_position => "beginning"
+  }
+#  stdin {}
+}
+
+filter {
+  grok {
+    match => { "message" => "%{TIMESTAMP_ISO8601:timestamp}\,%{BASE10NUM:duration} %{LOGLEVEL:level} %{DATA:category}\: %{GREEDYDATA:explain}"}
+  }
+}
+
+output {
+
+  kafka {
+   bootstrap_servers => "master1.valhalla.phdata.io:9093"
```

Comment:
Try to keep 'valhalla' out of the example configurations

Reply:
Safwan have you had this running on the cluster?

Reply:
@afoerster No I only had it running when testing out logstash